contracts/modules/cover/Cover.sol

```diff
@@ -647,6 +647,16 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
     }
   }
 
+  function editProducts(
```

Comment:
caveats 

Still have a separate call to change the IPFS Metadata hashes.



Reasoning: i made them separate because if i'm not touching the metadata value I don't want an event emitted.

Caveat 2


All fields are editable this way. In theory some fields may not be editable, like initialPrice, but this implementation is easier to make.

Otherwise I'd need to make an ProductEditChanges struct. I can see the idea of it but given this is AB controlled i don't see a big benefit.